jsk_2016_01_baxter_apc/launch/main_rgv5.launch

```diff
@@ -0,0 +1,155 @@
+  <group ns="left_hand">
+    <node name="object_verification"
+          pkg="jsk_2016_01_baxter_apc" type="apply_bin_contents_hint.py"
+          respawn="true">
+      <remap from="~input" to="vgg16_object_recognition/output" />
+      <param name="json" value="$(arg json)" />
+    </node>
+  </group>
+  <group ns="right_hand">
+    <node name="object_verification"
+          pkg="jsk_2016_01_baxter_apc" type="apply_bin_contents_hint.py"
+          respawn="true">
+      <remap from="~input" to="vgg16_object_recognition/output" />
+      <param name="json" value="$(arg json)" />
+    </node>
+  </group>
+  -->
+
+  <!-- robot main -->
+  <!--<node if="$(arg launch_main)" pkg="jsk_2016_01_baxter_apc" type="main-rgv5.l" name="left_hand"
+        args="&quot;(progn (apc-init :larm-controller) (apc-mainloop :larm))&quot;"
+        respawn="true" output="screen">
+    <env name="DISPLAY" value="" />
+    <rosparam subst_value="true">
+      collect_data: $(arg collect_data)
+    </rosparam>
+  </node>-->
+  <node if="$(arg launch_main)" pkg="jsk_2016_01_baxter_apc" type="main-rgv5.l" name="right_hand"
+    args="&quot;(progn (apc-init :rarm-controller) (apc-mainloop :rarm))&quot;"
+    respawn="true" output="screen">
+    <env name="DISPLAY" value="" />
+    <rosparam subst_value="true">
+      collect_data: $(arg collect_data)
+    </rosparam>
+  </node>
+
+  <!-- visualization -->
+  <node name="visualize_json"
+        pkg="jsk_apc2016_common" type="visualize_pick_json.py"
+        args="$(arg json)" />
+  <!--
+  <node name="visualize_on_rviz"
+        pkg="jsk_2015_05_baxter_apc" type="visualize_on_rviz.py">
+    <param name="json" value="$(arg json)" />
+  </node>
+  -->
+
+  <!-- collecting data for sib -->
+  <param name="/apc2016_collect_data" value="$(arg collect_data)" />
+  <group if="$(arg collect_data)">
+    <node name="data_collection_server_left"
+          pkg="jsk_data" type="data_collection_server.py">
+      <rosparam>
+        save_dir: ~/.ros/jsk_2016_01_baxter_apc/robocup2016
+        topics:
+          - name: /apply_mask_left_hand/output
+            msg_class: sensor_msgs/Image
+            savetype: ColorImage
+            fname: image.png
+          - name: /apply_mask_left_hand/output/mask
+            msg_class: sensor_msgs/Image
+            savetype: ColorImage
+            fname: mask.png
+      </rosparam>
+    </node>
+    <node name="data_collection_server_right"
+          pkg="jsk_data" type="data_collection_server.py">
+      <rosparam>
+        save_dir: ~/.ros/jsk_2016_01_baxter_apc/robocup2016
+        topics:
+          - name: /apply_mask_right_hand/output
+            msg_class: sensor_msgs/Image
+            savetype: ColorImage
+            fname: image.png
+          - name: /apply_mask_right_hand/output/mask
+            msg_class: sensor_msgs/Image
+            savetype: ColorImage
+            fname: mask.png
+      </rosparam>
+    </node>
+  </group>
```

Comment:
I think this data collection is not for current image pipeline. @wkentaro What do you think?